### PR TITLE
Adding more events

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,13 +336,15 @@ All the data (past and coming) are available publicly in JSON:
 * 25: [Voxxeddays Zurich 2025](https://zurich.voxxeddays.com/) - Zurich (Switzerland) <a href="https://vdz25.cfp.dev/#/login"><img alt="VoxxedDays Zürich 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%202-December-2024&color=green"></a>
 * 25: [ParisTestConf](https://paristestconf.com/) - Paris (France) <a href="https://sessionize.com/paristestconf-6/"><img alt="CFP ParisTestConf 6" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-November-2024&color=green"></a>
 * 26: [Rustikon 2025](https://www.rustikon.dev) - Warsaw (Poland) <a href="https://sessionize.com/rustikon-2025"><img alt="CFP Rustikon 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2013-October-2024&color=red"></a>
+* 26: [The DEVOPS Conference London 2025](https://www.thedevopsconference.com/) - London (UK) <a href="https://sessionize.com/the-devops-conference-london-2025"><img alt="CFP The DEVOPS Conference London 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2022-November-2024&color=green"></a>
```
Comment (scraly) [inline README.md:339]:
```suggestion
* 26: [The DEVOPS Conference London 2025](https://www.thedevopsconference.com/london) - London (UK) <a href="https://sessionize.com/the-devops-conference-london-2025"><img alt="CFP The DEVOPS Conference London 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2022-November-2024&color=green"></a>
```

File: README.md [hunk 2]
```diff
@@ -382,13 +392,16 @@ All the data (past and coming) are available publicly in JSON:
 ### May
 
 * 2: [Stir Trek 2025](https://stirtrek.com) - Columbus, OH (USA)
+* 4-7: [Leaders In Tech 2025](https://www.leaders-in.tech/) - Tel Aviv (Israel) <a href="https://sessionize.com/leaders-in-tech25/"><img alt="CFP Leaders In Tech 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-December-2024&color=green"></a>
 * 5-6: [GopherCon Latam 2025](https://gopherconlatam.org) - Florianópolis (Brazil) <a href="https://www.papercall.io/gopherconlatam2025"><img alt="CFP GopherCon Latam 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2006-November-2024&color=green"></a>
 * 7-9: [Devoxx UK](https://www.devoxx.co.uk/) - London (UK) <a href="https://devoxxuk25.cfp.dev/#/"><img alt="CFP devoxx uk 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2010-January-2025&color=green"></a>
+* 9: [Weblica 2025](https://weblica.hr/) - Štrigova (Croatia) <a href="https://sessionize.com/weblica-2025"><img alt="CFP Weblica 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2016-February-2025&color=green"></a>
```
Comment (scraly) [inline README.md:398]:
```suggestion
* 9: [Weblica 2025](https://weblica.hr/) - Strigova (Croatia) <a href="https://sessionize.com/weblica-2025"><img alt="CFP Weblica 2025" src="https://img.shields.io/static/v1?label=CFP&message=until%2016-February-2025&color=green"></a>
```

File: page/src/misc/geolocations.json [hunk 3]
```diff
@@ -2266,5 +2274,9 @@
   "ibenik (Croatia)": {
     "latitude": 43.7350196,
     "longitude": 15.8952045
+  },
+  "trigova (Croatia)": {
```
Comment (scraly) [inline page/src/misc/geolocations.json:2278]:
```suggestion
  "Strigova (Croatia)": {
```

**Review by scraly — CHANGES_REQUESTED** [review]
Thanks @jespino 
just few remarks :)

**Comment by jespino** [comment]
@scraly fixed, thanks!

**Comment by scraly** [comment]
thanks :)

**Pull request merged**